json-path/src/test/java/io/zeebe/msgpack/mapping/MappingMergeParameterizedTest.java

```diff
@@ -322,25 +322,6 @@
             // expected result
             "{'obj':{'test':'value','newFoo':1},'int':1}"
           },
-          // 37. Test zeebe-io/zeebe#297
```

Comment:
why :sweat_smile:

Reply:
I deleted the test case because it was outdated since a while :smile:

Reply:
ok :smile: